<launch>
  <arg name="device_name"             default="/dev/ttyUSB0"/>
  <arg name="baud_rate"               default="57600"/>

  <arg name="left_wheel"              default="1"/>
  <arg name="right_wheel"             default="2"/>

  <arg name="wheel_separation"        default="0.160"/> <!--Refer to TurtleBot3 Burger-->
  <arg name="wheel_radius"            default="0.033"/> <!--Refer to TurtleBot3 Burger-->

  <arg name="profile_velocity"        default="200"/>
  <arg name="profile_acceleration"    default="50"/>

  <param name="device_name"           value="$(arg device_name)"/>
  <param name="baud_rate"             value="$(arg baud_rate)"/>

  <param name="left_wheel"            value="$(arg left_wheel)"/>
  <param name="right_wheel"           value="$(arg right_wheel)"/>

  <param name="profile_velocity"      value="$(arg profile_velocity)"/>
  <param name="profile_acceleration"  value="$(arg profile_acceleration)"/>

  <node name="velocity_control" pkg="dynamixel_workbench_controllers" type="velocity_control" required="true" output="screen"/>
</launch>
